getting_hired/project_portfolio.md

```diff
@@ -14,6 +14,38 @@ Remember to write some documentation for both technical and non-technical audien
 
 * [A Beginner’s Guide to Writing Documentation](http://www.writethedocs.org/guide/writing/beginners-guide-to-docs/)
 
+### Design Inspirations
+<details markdown="block">
+  <summary> Links for designs </summary>
+
+  * [Stratis Bakas](https://stratisbakas.com/)
+  * [Matt Farley](https://mattfarley.ca/)
+  * [Dejan Markovic](https://www.dejan.works/)
+  * [Rafael Caferati](https://caferati.me/)
+  * [Ian Lunn](https://ianlunn.co.uk/)
+  * [Ben Adam](https://benadam.me/)
+  * [Seb Kay](https://sebkay.com/)
+  * [Jonny MacEachern](https://jonny.me/)
+  * [Jack Jeznach](jacekjeznach.com)
```

Comment:
This link in the markdown needs to be fixed.

Reply:
I have fixed this link.